src/calculateRentalCost.js

```diff
@@ -4,7 +4,9 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const off = days >= 7 ? 50 : days >= 3 ? 20 : 0;
```

Comment:
Avoid using literals directly in expressions. Instead, define constants for these values to make the code more readable and maintainable. For example, define constants for the daily rate, discount thresholds, and discount amounts. This aligns with the checklist item: 'Don't use literals in expressions (HARDCODE). Instead use constants explaining this values.'